specification/containerservices/resource-manager/Microsoft.ContainerService/preview/2018-09-30-preview/openShiftManagedClusters.json

```diff
@@ -0,0 +1,590 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create/Update OpenShift Managed Cluster": {
+            "$ref": "./examples/OpenShiftManagedClustersCreate_Update.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "OpenShiftManagedClusters"
+        ],
+        "operationId": "OpenShiftManagedClusters_UpdateTags",
+        "summary": "Updates tags on an openshift managed cluster.",
+        "description": "Updates an openshift managed cluster with the specified tags.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the openshift managed cluster resource."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to the Update OpenShift Managed Cluster Tags operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OpenShiftManagedCluster"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Update OpenShift Managed Cluster Tags": {
+            "$ref": "./examples/OpenShiftManagedClustersUpdateTags.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "OpenShiftManagedClusters"
+        ],
+        "operationId": "OpenShiftManagedClusters_Delete",
+        "summary": "Deletes an openshift managed cluster.",
+        "description": "Deletes the openshift managed cluster with a specified resource group and name.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the openshift managed cluster resource."
+          }
+        ],
+        "responses": {
```

Comment:
model the 200 response. 204 will indicate that the resource to be deleted was not found. and 200 will indicate that the resource was found and deleted successfully.

Reply:
Hi @ravbhatnagar we don't have a 200 response here, only 202 or 204. is it ok ?

Reply:
so as per ARM Resource provider contract, 204 on a delete must be returned when the requested resourceId was not found. 200 is returned in case where resource was successfully deleted.